acmw-db-app/pages/signin.js

```diff
@@ -96,24 +103,43 @@ const TextInput = (props) => {
         onFocus={() => setActive(true)}
         onBlur={() => setActive(false)}
       />
-      <label htmlFor={id} className={error && styles.error}>
-        {error || label}
-      </label>
```

Comment:
I think this was the little purple text that said "Username" (will be switched to "Email") and "Password" when the field was focused, so probably put back:
```
<label htmlFor={id}>	
    {label}
</label>
```

Reply:
oh right yeah my b